initial_pose: 
  x:  0.17289673141400325
  y:  2.4519335360095704
  z:  0.9734673165727412
  R_deg: 0
  P_deg: 0
  Y_deg: 0
  camera_deg: -82
collision_distance: 0.85